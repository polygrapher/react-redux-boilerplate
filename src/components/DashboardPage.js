import React from 'react';

const DashboardPage = () => (
    <div>
        <h1>Dashboard content</h1>
    </div>
);

export default DashboardPage;