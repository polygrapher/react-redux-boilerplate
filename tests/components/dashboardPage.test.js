import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../src/components/DashboardPage';

test('should render dashboard page', () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
});