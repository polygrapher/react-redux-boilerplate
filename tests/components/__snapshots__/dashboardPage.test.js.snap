// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page 1`] = `
<div>
  <h1>
    Dashboard content
  </h1>
</div>
`;
